form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(),Email()])
```

Comment:
I'd say to add a space between the validators per [PEP-8](http://legacy.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).